### ISSUE TITLE
Diffuse breast nodularity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10110]
**Preferred term label:**
Diffuse breast nodularity

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0741750

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lobular breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000069
- label: Diffuse breast nodularity
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: 
- def: Breast tissue with a lumpy or ropelike texture.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10110
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
corrected HPO id to  HP:0040415
